sdk/communication/Azure.Communication.PhoneNumbers/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2021-03-09)
```

Comment:
Does Phone Numbers have a 2021-03-09 release date as well? I thought SMS is being released later.

Reply:
From what I understand we are releasing everything this time around.. let me confirm once again